File: yazi/.config/yazi/flavors/eldritch.yazi/.theme/theme.swift
// comment.line.double-slash
/* comment.block */

/// comment.documentation

let num: Int = 42                 // constant.numeric
let esc: Character = "\n"         // constant.character.escape
let flag: Bool = true             // constant.language

class MyClass: BaseClass {        // entity.name.type, entity.inherited-class
    func myFunction(param: String) -> String { // entity.name.function, variable.parameter
        let color = "#ff0000"     // constant.other, string.quoted.double
        if flag {                 // keyword.control
            return color          // keyword.control
        }
        return ""
    }
}

File: yazi/.config/yazi/flavors/eldritch.yazi/.theme/theme.cpp
// comment.line.double-slash
/* comment.block */
/// comment.documentation

const int NUM = 42;     // constant.numeric
const char ESC = '\n';  // constant.character.escape
const bool FLAG = true; // constant.language

class MyClass : public BaseClass { // entity.name.type, entity.inherited-class
public:
  static void func(
      int param) { // entity.name.function, variable.parameter, storage.modifier
    int result = NUM + 1; // variable.other
    return;               // keyword.control
  }
};

#define COLOR "#ff0000" // constant.other
if (FLAG) {             // keyword.control
  result = a || b;      // keyword.operator
}
